README.md

```diff
@@ -1,2 +1,24 @@
 # flytekit-python-template
 A template for the recommended layout of a Flyte enabled repository for code written in python using flytekit
+
+## Usage
+To get up and running with your Flyte project, we recommend proceeding through the following:
+
+* Deploy the flyte sandbox locally or to any Kubernetes Cluster (EKS, GKE, custom, etc) by following the instructions at
+   the [getting started guide](https://flyte.readthedocs.io/en/latest/administrator/install/getting_started.html#getting-started)
+* Check the status of your [local deployment](http://localhost/console) and assert that it contains sample projects or
+   in the case of a remote deployment enable port-forwarding
+   
+```shell
+kubectl -n flyte port-forward svc/envoy 8001:80
```

Comment:
sorry one more comment. We need to port-forward if the hosted environment is not publicly accessible.
Otherwise, you can just access it using the external IP as described here
https://github.com/flyteorg/flyte/pull/683

Reply:
sure, done